rs/tests/message_routing/xnet/BUILD.bazel

```diff
@@ -100,6 +100,7 @@ system_test_nns(
         "XNET_TEST_CANISTER_WASM_PATH": "$(rootpath //rs/rust_canisters/xnet_test:xnet-test-canister)",
     },
     extra_head_nns_tags = ["manual"],  # only run this test with the mainnet NNS canisters.
+    flaky = True,
```

Comment:
I would recommend disabling this test by marking it as `manual` and asking the owners to fix the flakiness.